docusaurus/docs/protocol/primitives/claim_and_proof_lifecycle.md

```diff
@@ -714,7 +723,7 @@ stateDiagram-v2
 
 [*] --> Validate_Relay_Request
 state Validate_Relay_Request {
```

Comment:
I did a ctrl-f for `supplier_addr` and found a few outstanding ones:

![Screenshot 2024-08-07 at 12 59 33 PM](https://github.com/user-attachments/assets/2c3a1f14-7699-4b8d-b5d6-0c1afad958a3)